core/src/main/java/com/exadel/aem/toolkit/bundle/customlists/datasource/ItemComponentDatasource.java

```diff
@@ -52,32 +53,39 @@
     }
 )
 public class ItemComponentDatasource extends SlingSafeMethodsServlet {
+    private static final Logger LOG = LoggerFactory.getLogger(ItemComponentDatasource.class);
+
     private static final String SELECT_STATEMENT = "SELECT * FROM [cq:Component] AS s WHERE ISDESCENDANTNODE(s,'/apps') AND [componentGroup] = 'acl-item-component'";
 
     private static final String VALUE = "value";
```

Comment:
`PN_TEXT`? Or could it still be a common constants class possibly?

Reply:
Agreed to think about common constants class when CustomDatasource class is moved into toolkit